packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -141,6 +151,7 @@ class CupertinoTextField extends StatefulWidget {
     this.decoration = _kDefaultRoundedBorderDecoration,
     this.padding = const EdgeInsets.all(6.0),
     this.placeholder,
+    this.placeholderStyle = _kDefaultPlaceholderStyle,
```

Comment:
Thanks for the PR! Sorry for the slow response since a lot of us were out during the holidays.

My general feedback is that we should keep this pattern consistent with the other argument `style` where the default is null (where it then pull the style from the ancestor theme), otherwise, the supplied override is used. 

We can do the same thing here and let `placeholderStyle` be defaulted to null.